fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -60,18 +60,49 @@
   </xsl:template>
 
 
+  <!-- the following template creates an _ms field for thesis advisors -->
```

Comment:
Ready for review and merge.
See Jira TRAC-886 for screenshots of my tests.
TRACE(TRAC-886) loads utk-fedora-fedoragsearch-solr-confif(TRAC-886a) 
@markpbaggett @CanOfBees

Reply:
@cdeaneGit, reviewing now.